posthog/session_recordings/queries/test/test_session_recording_list_from_filters.py

```diff
@@ -809,10 +809,7 @@ def test_filter_on_session_ids(self):
             }
         )
 
-        assert sorted(
-            [s["session_id"] for s in session_recordings],
-            key=lambda x: x[0],
```

Comment:
the fun thing here (I don't remember why the code is this shape). is this is sorting the strings based on their first character so every so often the test would fail just because the first character sorting and the lexical string sorting didn't match 🙈

Reply:
ah, it was copy pasta... cleared up several tests with the same mistake